docs/BBC-BASIC-for-Agon.md

```diff
@@ -111,7 +111,7 @@ Change the the current text output colour
 - If c is between 0 and 63, the foreground text colour will be set
 - If c is between 128 and 191, the background text colour will be set
 
-The following two commands are only applicable to paletted modes with less than 64 colours.
+The following two commands are only applicable to palette modes with less than 64 colours.
 
```

Comment:
```suggestion
```
as per previous comment, suggestion here is to delete this line, for simplicity

we could alternatively replace this with something more complicated to explain which VDP version it was where this command would start working in 64 colour modes, but that feels overly complex for this particular file

Reply:
Good point - I'll remove the line for simplicity